src/App.jsx

```diff
@@ -1,14 +1,22 @@
 import React from "react";
+import { 
+  BrowserRouter as Router,
```

Comment:
I don't think it's necessary to import BrowserRouter here, it's in index.js that you use it.